### PR TITLE
Editorial Review - Community

### PR DESCRIPTION
**Pull request by SandraGH5**
Reshuffled top buttons, and added buttons for Newsletter and Wednesday office hours.

### CODE REVIEW

File: rsts/community/index.rst [hunk 1]
```diff
@@ -5,8 +5,8 @@ Community
 ##########
 
 Flyte is an ambitious open source project and would not be possible without an
-amazing community. We are a completely open community and we vouch to treat
-every member with respect. You will find the community very welcoming and warm,
+amazing community. We are a completely open community and vouch to treat
```
Comment (cosmicBboy) [inline rsts/community/index.rst:8]: vouch -> strive?

File: rsts/community/index.rst [hunk 2]
```diff
@@ -85,12 +93,27 @@ If you'd like to give a 3-5 minute ⚡ Lightning Talk ⚡ during our Open Source
...
     :classes: btn-outline-secondary
 
+Office Hours
+------------
+
+Flyte maintainers are available on Zoom for you to ask anything, weekly on Wednesdays:
+
+.. link-button:: https://www.addevent.com/event/zF10349020/
+    :type: url
+    :text: 7:00 am PT / 3:00 pm GMT - Haytham Abuelfutuh
```
Comment (cosmicBboy) [inline rsts/community/index.rst:103]: let's future-proof this by removing the name, so just `7:00 am PT / 3:00 pm GMT`

File: rsts/community/index.rst [hunk 3]
```diff
@@ -85,12 +93,27 @@ If you'd like to give a 3-5 minute ⚡ Lightning Talk ⚡ during our Open Source
...
+    :classes: btn-outline-secondary
+
+.. link-button:: https://www.addevent.com/event/dQ10349168/
+    :type: url
+    :text: 9:00 pm PT / 10:30 am IST - Ketan Umare
```
Comment (cosmicBboy) [inline rsts/community/index.rst:108]: same here: `9:00 pm PT / 10:30 am IST`

File: rsts/community/index.rst [hunk 4]
```diff
@@ -85,12 +93,27 @@ If you'd like to give a 3-5 minute ⚡ Lightning Talk ⚡ during our Open Source
...
+    :classes: btn-outline-secondary
+
 .. toctree::
     :caption: Community
     :maxdepth: -1
     :name: communitytoc
     :hidden:
- 
+
```
Comment (cosmicBboy) [inline rsts/community/index.rst:116]: remove empty space
```suggestion

```

File: rsts/community/index.rst (hunk 1 above)
Comment (SandraGH5) [reply rsts/community/index.rst:8]: Cool.

File: rsts/community/index.rst (hunk 2 above)
Comment (SandraGH5) [reply rsts/community/index.rst:103]: Good idea.

File: rsts/community/index.rst (hunk 3 above)
Comment (SandraGH5) [reply rsts/community/index.rst:108]: Same.

**Review by SandraGH5 — COMMENTED** [review]
Suggestions accepted.

**Comment by SandraGH5** [comment]
Are we able to merge now @cosmicBboy?

**Review by kumare3 — APPROVED** [review]

**Pull request merged**